### PR TITLE
Localization/Variant fixes, MH part hide patch fix, MH RCS Mini config fix

### PR DESCRIPTION
**Pull request by Poodmund**
- Cleaned up some localization tags and variant ordering/naming
- Fixed MH Part hiding patch
- Fixed MH RCS Mini Tank part name in config

### CODE REVIEW

File: Distribution/RestockPlus/GameData/ReStockPlus/Localization/en-us.cfg [hunk 1]
```diff
@@ -142,7 +142,7 @@ Localization
     #LOC_RestockPlus_restock-fairing-base-0625-1_description = While the Kerbals at Mission Control were still figuring out how to get their rockets back down to Kerbin safely, the research engineers at FLOOYD were quickly realising that protecting parts on ascent was just as important. Heavy research into two-dimensional-input driven procedural construction was then funded with the hopes of making protective shells for important payloads and interstage areas of the crafts. The protective shells also have the benefit of making the craft more aerodynamic, hopefully saving on precious rocket fuel! The AE-FF0 is an even tinier size available from FLOOYD.
     #LOC_RestockPlus_restock-fairing-base-0625-1_tags = restock aero )cap cargo cone contain drag fairing hollow inter nose payload protect rocket shroud stage (stor transport
 
-    #LOC_RestockPlus_restock-fairing-base-1875-1_title = AE-FF1-L Airstream Protective Shell (1.875m)
+    #LOC_RestockPlus_restock-fairing-base-1875-1_title = AE-FF1.5 Airstream Protective Shell (1.875m)
```
Comment (ChrisAdderley) [inline Distribution/RestockPlus/GameData/ReStockPlus/Localization/en-us.cfg]: Let's not replicate the MH part name exactly

File: Distribution/Restock/GameData/ReStock/Patches/Engine/restock-engines-srb-125.cfg [hunk 2]
```diff
@@ -397,14 +397,12 @@
...
   {
-    baseVariant = White
-    !primaryColor = DELETE
-    !secondaryColor = DELETE
-    !baseDisplayName = DELETE
-    !baseThemeName = DELETE
-    useMultipleDragCubes = false
+    name = ModulePartVariants
+	baseVariant = White
+	useMultipleDragCubes = false
```
Comment (ChrisAdderley) [inline Distribution/Restock/GameData/ReStock/Patches/Engine/restock-engines-srb-125.cfg]: Whitespace

File: Distribution/Restock/GameData/ReStock/PatchesMH/FuelTank/restock-mh-fueltanks-rcs.cfg [hunk 3]
```diff
@@ -3,7 +3,7 @@
 // -  (MonoPropMini)
 
 //  Spherical Radial Tiny Tank
-@PART[MonoPropMini]
+@PART[monopropMiniSphere]
```
Comment (ChrisAdderley) [inline Distribution/Restock/GameData/ReStock/PatchesMH/FuelTank/restock-mh-fueltanks-rcs.cfg]: Ah Lupi led me wrong!

File: Distribution/Restock/GameData/ReStock/PatchesMH/Payload/restock-mh-fairing-bases.cfg [hunk 4]
```diff
@@ -21,9 +21,9 @@
   @MODULE[ModuleProceduralFairing]
   {
     @pivot = 0,0.151, 0
-    @TextureURL = ReStock/Assets/Payload/restock-fairing-25-1
+    @TextureURL = ReStock/Assets/Payload/restock-fairing-125-1
     @CapTextureURL = ReStock/Assets/Payload/restock-fairing-cap-1
-    @TextureNormalURL = ReStock/Assets/Payload/restock-fairing-25-1-n
+    @TextureNormalURL = ReStock/Assets/Payload/restock-fairing-125-1-n
```
Comment (ChrisAdderley) [inline Distribution/Restock/GameData/ReStock/PatchesMH/Payload/restock-mh-fairing-bases.cfg]: You're certain this looks better?

File: Distribution/RestockPlus/GameData/ReStockPlus/Localization/en-us.cfg (hunk 1 above)
Comment (Poodmund) [reply Distribution/RestockPlus/GameData/ReStockPlus/Localization/en-us.cfg]: Ah okay, I thought the idea was for it to be the same. No worries.

File: Distribution/Restock/GameData/ReStock/Patches/Engine/restock-engines-srb-125.cfg (hunk 2 above)
Comment (Poodmund) [reply Distribution/Restock/GameData/ReStock/Patches/Engine/restock-engines-srb-125.cfg]: Yeah there are inconsistencies here. To be resolved at later time

**Pull request merged**